server.js

```diff
@@ -4759,6 +4759,40 @@ cache(function(data, match, sendBadge, request) {
   sendBadge(format, badgeData);
 }));
 
+// homebrew integration
+camp.route(/^\/homebrew\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Could you make it `/homebrew/v/PACKAGE.FORMAT` instead? The addition of the `/v/` allows more extensibility in the future, in case there are more badges made around Homebrew.